spec/services/save_provider_user_notification_preferences_spec.rb

```diff
@@ -54,5 +54,30 @@
         )
       end
     end
+
+    context 'when the :make_decision_reminder_notification_setting feature flag is off' do
+      before do
+        FeatureFlag.deactivate(:make_decision_reminder_notification_setting)
+      end
+
+      let(:notification_preferences_params) do
+        {
+          application_received: true,
```

Comment:
this test will pass if we remove the implementation as the defaults are true, could we set this to false instead to ensure the test will fail if we don't have the implementation above?

Reply:
Sorted, it's now the other way around